armeria-backend/zio/src/main/scala/sttp/client3/armeria/zio/ArmeriaZioBackend.scala

```diff
@@ -49,11 +49,13 @@ object ArmeriaZioBackend {
       .runtime[Any]
       .map(runtime => apply(runtime, newClient(options), closeFactory = true))
 
-  def managed(options: SttpBackendOptions = SttpBackendOptions.Default): TaskManaged[SttpBackend[Task, ZioStreams]] =
-    ZManaged.acquireReleaseWith(apply(options))(_.close().ignore)
+  def managed(
```

Comment:
Does it have sense calling it managed? Maybe rename it to scoped?

Reply:
I had the same thought. Happy to change it if we think that's a better name.

Reply:
Softwaremill guys make a decision and let me know what you want to see here. Happy to make the changes or leave it as is.